galaxy.yml

```diff
@@ -2,7 +2,7 @@
 
 namespace: community
 name: docker
-version: 2.6.0
+version: 3.0.0-alpha.0
```

Comment:
This must be removed from this PR. Release-related changes are not part of regular PRs.